frontend/src/app/core/env.config.ts

```diff
@@ -0,0 +1,13 @@
+const isDev = window.location.port.indexOf('4200') > -1;
```

Comment:
Looks fine. But not sure if Angular is able to inject environment variables like webpack does, in which case you could just use `process.env.NODE_ENV !== 'production'`